src/scss/components/file/_index.scss

```diff
@@ -0,0 +1,65 @@
+@import "components/form/config";
+@import "components/box/index";
```

Comment:
This is inlining `box` styles into this CSS.  Import the config, instead.

```scss
@import "components/box/config";
```

Reply:
Additionally, you'll want to import mixins only for the files that actually need them.

* `components/form/config` for `_hx-file-control.scss`
* `components/box/config` for `_hx-drop-fence.scss`
* etc.

Reply:
Ok, will do @CITguy.

Also, we need to document the new SCSS file structure to help the team and future contributors.

Reply:
added `@import "components/box/config"`